ci/run.sh

```diff
@@ -26,6 +26,7 @@ case ${TARGET} in
         export STDSIMD_DISABLE_ASSERT_INSTR=1
         ;;
     powerpc64-*)
+        export RUSTFLAGS="${RUSTFLAGS} -C target-feature=+altivec"
```

Comment:
This should not be required

Reply:
That is some stray from testing, let me remove it now.